lib/fathead/sass/parse_functions.py

```diff
@@ -229,18 +262,19 @@ def create_file(self):
                     method_signature = data_element.get('method_signature').encode('utf-8')
                     description = data_element.get('description').encode('utf-8')
                     name = data_element.get('function').encode('utf-8')
+
                     abstract = str(method_signature) + '<br>' + str(description)
                     url = data_element.get('url').encode('utf-8')
                     list_of_data = [
                         name,                       # unique name
                         'A',                        # type is article
                         '',                         # no redirect data
                         '',                         # ignore
-                        '',                         # no categories
+                        'functions',                # no categories
```

Comment:
Also, if it **is** a function, I think we should build a redirect to the current article, by combining `$article_title function"

That was "sass rgba function" will redirect to the "rgba" article and display: https://beta.duckduckgo.com/?q=sass+rgba&t=hc&ia=about

Right now, that query doesn't work: https://duckduckgo.com/?q=sass+rgba+function&t=hc&ia=qa

With the categories, "sass functions" will produce a "Meanings" ew with tiles representing all the articles we've put into the "sass functions" category :+1: